lisa/testsuite.py

```diff
@@ -91,6 +93,10 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
         self._timer: Timer
 
         self._environment_information: Dict[str, Any] = {}
+        parent_log = get_logger("suite", self.runtime_data.metadata.suite.name)
+        self.log = get_logger("case", self.name, parent=parent_log)
+        self.case_log_path = self.__create_case_log_path()
```

Comment:
The log path shouldn't be created at this memont, because the timestamp of the path indicates when the case started.

Reply:
Changed, now it is created when path is read for the first time. Is that fine?